share/picongpu/examples/LaserWakefield/lib/python/picongpu/params.py

```diff
@@ -36,6 +39,12 @@
 
     LinearScaledParameter(
         name="Pulse_Length_SI", ptype="compile", unit="s",
-        default=5, slider_min=1, slider_max=150,
+        default=5.0, slider_min=1.0, slider_max=150.0,
         slider_step=1, scale_factor=1.e-15),
+
+    LinearScaledParameter(
+        name="TBG_steps", ptype="run", unit="s",
```

Comment:
For the `Wave_Length_SI` parameter the default should be a float or the dtype should be specified as float, since right now the dtype is taken as `int` causing much trouble in the computations that follow.